src/components/SignerPanel/ActionPathsContent.js

```diff
@@ -83,7 +83,7 @@ class ActionPathsContent extends React.Component {
                         mode: {
                           name: 'org',
                           instanceId: 'permissions',
-                          params: `app.${value.proxyAddress}`,
+                          instancePath: `/app/${value.proxyAddress}`,
```

Comment:
We must've forgotten to update this in the last PR 😅

Reply:
Good catch 😅

Reply:
I realize now that it is the issue discussed here https://github.com/aragon/aragon/pull/1457/files#r448042918

With the removal of the `extend` param, the idea would have been to extend `mode` ourselves here:

```jsx
routing.path(({ mode }) => ({
  mode: {
    ...mode,
    instanceId: 'permissions', 
    instancePath: `/app/${value.proxyAddress}`, 
  }
})
```